kubernetes/generic/bookie.yaml

```diff
@@ -57,7 +57,7 @@ spec:
         spec:
             containers:
               - name: bookie
-                image: streamlio/pulsar:latest
+                image: apachepulsar/pulsar:latest
```

Comment:
Then we would need to manually change it at every release. 
Of course in a prod deployment you want to set a specific version. In this case, it's easier to just make it pick the latest stable version.